code/modules/buildmode/submodes/link.dm

```diff
@@ -46,39 +46,39 @@
 			if(!M.id || M.id == "")
 				M.id = input(user, "Please select an ID for the button", "Buildmode", "")
 				if(!M.id || M.id == "")
-					goto(line_jump)
+					goto line_jump
```

Comment:
http://www.byond.com/docs/ref/index.html#/proc/goto
goto should not be called like a proc.